tutorials/image/cifar10/cifar10_multi_gpu_train.py

```diff
@@ -169,7 +176,7 @@ def train():
             # Calculate the loss for one tower of the CIFAR model. This function
             # constructs the entire CIFAR model but shares the variables across
             # all towers.
-            loss = tower_loss(scope)
+            loss = tower_loss(scope, images, labels)
```

Comment:
I made a few cleanup changes. Nice that testing showed it does better than previously.

Is slim the best way to get this to work? This is a tutorial folder so it would be ideal to avoid having slim code here (and contrib code in general, if we can avoid it).